docs/installation/3-netbox.md

```diff
@@ -267,6 +267,13 @@ Starting development server at http://0.0.0.0:8000/
 Quit the server with CONTROL-C.
 ```
 
+!!! note
+    By default RHEL based distros will likely block your testing attempts with firewalld. The development server port can be opened with `firewall-cmd` (add `--permanent` if you want the rule to survive server restarts):
+
+```no-highlight
```

Comment:
This block needs to be indented to be displayed within the note.

Reply:
Done, do you want me to squash the commits or you'll squash on merge?

Reply:
:+1: Thanks, I'll squash on merge.